reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+editor on Windows, such as Visual Studio Code, results in a file encoded using
+`UTF8NoBOM`. These files work fine on PowerShell Core, but may break on Windows
+PowerShell if the file contains non-Ascii characters.
+
+If you do need to use non-Ascii characters in your scripts, save them as UTF-8
+with BOM. Without the BOM, Windows PowerShell misinterprets your script as
+being encoded in the legacy "ANSI" codepage. Conversely, files that do have the
+UTF-8 BOM can be problematic on Unix-like platforms. Many Unix tools such as
+`cat`, `sed`, `awk`, and some editors such as `gedit` don't know how to treat
+the BOM.
+
+## Character encoding in Windows PowerShell
+
+In PowerShell 5.1, the **Encoding** parameter supports the following values:
+
+- `Ascii` Uses Ascii (7-bit) character set.
+- `BigEndianUnicode` Uses UTF-16 with the big-endian byte order.
+- `BigEndianUTF32` Uses UTF-32 with the big-endian byte order.
+- `Byte` Encodes a set of characters into a sequence of bytes.
+- `Default` Uses the encoding that corresponds to the system's active code page
+  (usually ANSI).
+- `Oem` Uses the encoding that corresponds to the system's current OEM code
+  page.
+- `String` Same as `Unicode`.
+- `Unicode` Uses UTF-16 with the little-endian byte order.
+- `Unknown` Same as `Unicode`.
+- `UTF7` Uses UTF-7.
+- `UTF8` Uses UTF-8.
+- `UTF32` Uses UTF-32 with the little-endian byte order.
+
+In general, Windows PowerShell uses the Unicode
+[UTF-16LE](https://en.wikipedia.org/wiki/UTF-16) encoding by default. However,
+the default encoding used by cmdlets in Windows PowerShell is not consistent.
+
+> [!NOTE]
+> Using any Unicode encoding, except `Utf7`, always creates a BOM.
+
+For cmdlets that write:
+
+- `Out-File` and the redirection operators `>` and `>>` create UTF-16LE, which
+  notably differs from `Set-Content` and `Add-Content`.
+
+- `New-ModuleManifest` and `Export-CliXml` also create UTF-16LE files.
+
+- When the target file is empty or doesn't exist, `Set-Content` and
+  `Add-Content` use `Default` encoding. `Default` is the encoding specified by
+  the active system locale's ANSI legacy code page.
+
+- `Export-Csv` creates `Ascii` files but uses different encoding when using
+  **Append** parameter (see below).
+
+- `Export-PSSession` creates UTF-8 files with BOM by default.
+
+- `New-Item -Type File -Value` creates a BOM-less UTF-8 file.
+
+- `Send-MailMessage` uses `Default` encoding by default.
+
+- `Start-Transcript` creates `Utf8` files with a BOM. When the **Append**
+  parameter is used, the encoding can be different (see below).
+
+For commands that append to an existing file:
+
+- `Out-File -Append` and the `>>` redirection operator make no attempt to match
+  the encoding of the existing target file's content. Instead, they use the
+  default encoding unless instructed the **Encoding** is used. You must know
+  the encoding of an existing file's content to append using the same encoding.
```

Comment:
I'd also rephrase this sentence.  Maybe just "You must use the files original encoding when appending content." or something along those lines.

Reply:
Fixed